shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/ApiServiceImpl.java

```diff
@@ -110,6 +109,7 @@ public String createOrUpdate(final ApiDTO apiDTO) {
      * @return update message
      */
     private String update(final ApiDTO apiDTO) {
+        //cdbuild apiDO
```

Comment:
why add this？

Reply:
Hello, my previous submission failed to check the instability of ci. I hope I can re-trigger ci through some modifications. If these modifications cause bad effects, I will delete them